### PR TITLE
Add release notes for 5.17

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Add release notes to support 5.17 release.

### CODE REVIEW

File: content/sensu-go/5.17/release-notes.md [hunk 1]
```diff
@@ -50,6 +51,30 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+## 5.17.0 release notes
+
+**January 23, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
+PLACEHOLDER FOR RELEASE SUMMARY.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.17.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][106]) Added [HTTP API for secrets provider management][108], and a built-in `Env` secrets provider. The secrets provider resource is implemented for checks, mutators, and handlers.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
- ([Commercial feature][106]) Added [HTTP API for secrets provider management][108] and a built-in `Env` secrets provider. The secrets provider resource is implemented for checks, mutators, and handlers.
```

File: content/sensu-go/5.17/release-notes.md [hunk 2]
```diff
@@ -50,6 +51,32 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- In the [web UI][107], labels that contain URLs are now clickable links.
+
+**FIXES:**
+
+- ([Commercial feature][106]) Fixed a bug where the event check state was not present when using the PostgreSQL event store.
+- Fixed a memory leak in the entity cache.
+- Initialization via [`sensu-backend-init`][109] is now implemented for Docker.
+- Fixed a bug that prevented `sensuctl entity delete` from returning an error when attempting to delete a non-existent entity.
+- In the [web UI][107], fixed a bug that duplicated event history in the event timeline chart.
+- Sensuctl command assets installed via Bonsai now use the `sensuctl` namespace.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
- `sensuctl command` assets installed via Bonsai now use the `sensuctl` namespace.
```

File: content/sensu-go/5.17/release-notes.md [hunk 3]
```diff
@@ -50,6 +51,32 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.17.0 release notes
+
+**January 23, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
**January 27, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
```

File: content/sensu-go/5.17/release-notes.md [hunk 4]
```diff
@@ -50,6 +51,32 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+## 5.17.0 release notes
+
+**January 27, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
+PLACEHOLDER FOR RELEASE SUMMARY.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.17.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][106]) Added [HTTP API for secrets provider management][108] and a built-in `Env` secrets provider. The secrets provider resource is implemented for checks, mutators, and handlers.
```
Comment (echlebek) [inline content/sensu-go/5.17/release-notes.md]: We should mention that we also support Vault for secrets management.

File: content/sensu-go/5.17/release-notes.md [hunk 5]
```diff
@@ -50,6 +51,32 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.17.0 release notes
+
+**January 27, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
**January 28, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
```

File: content/sensu-go/5.17/release-notes.md (hunk 4 above)
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
- ([Commercial feature][106]) Added [HTTP API for secrets management][108], with a built-in `Env` secrets provider and support for HashiCorp Vault secrets management. The secrets provider resource is implemented for checks, mutators, and handlers.
```

**Comment by portertech** [comment]
Adding release synopsis, updating the branch momentarily.

**Review by echlebek — APPROVED** [review]
L G T M
G T M L
T M L G
M L G T

File: content/sensu-go/5.17/release-notes.md [hunk 6]
```diff
@@ -53,9 +53,7 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ## 5.17.0 release notes
 
-**January 28, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
-
-**PLACEHOLDER FOR RELEASE SUMMARY.**
+**January 28, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download. This is a significant release, with new features, improvements, and fixes! We’re ecstatic to announce the release of secrets management, which eliminates the need to expose sensitive information in your Sensu configuration. When a Sensu component (e.g. check, handler, etc.) requires a secret (like a username or password), Sensu will be able to fetch that information from one or more external secrets providers (e.g. Hashicorp Vault) and provide it to the Sensu component via temporary environment variables. By leveraging secrets management, you’re able to move secrets out of your Sensu configuration, giving you the ability to safely, easily, and confidently share your Sensu configurations with your fellow Sensu users! This release also includes per entity keepalive event handler configuration, a sought after feature for users who have migrated from Sensu 1.x to Sensu Go.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
**January 28, 2020** &mdash; The latest release of Sensu Go, version 5.17.0, is now available for download.
This is a significant release, with new features, improvements, and fixes!
We’re ecstatic to announce the release of secrets management, which eliminates the need to expose sensitive information in your Sensu configuration.
When a Sensu component (e.g. check, handler, etc.) requires a secret (like a username or password), Sensu will be able to fetch that information from one or more external secrets providers (e.g. HashiCorp Vault) and provide it to the Sensu component via temporary environment variables.
Secrets management allows you to move secrets out of your Sensu configuration, giving you the ability to safely and confidently share your Sensu configurations with your fellow Sensu users!
This release also includes per-entity keepalive event handler configuration, a sought-after feature for users who have migrated from Sensu 1.x to Sensu Go.
```

**Review by ccressent — APPROVED** [review]

**Pull request merged**